frontend/src/ts/kiosk.ts

```
@@ -151,20 +152,43 @@ function handleFullscreenClick(): void {
-      e.preventDefault();
-      togglePolling();
+
+    switch (e.code) {
+      case "Space":
+        e.preventDefault();
+        togglePolling(true);
+        break;
+      case "KeyI":
+        e.preventDefault();
+        const isPollingPaused =
+          document.body.classList.contains("polling-paused");
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)